lib/unleash/util/event_source_wrapper.rb

```diff
@@ -0,0 +1,17 @@
+module Unleash
+  module Util
+    module EventSourceWrapper
+      def self.client
```

Comment:
Hmm, I'm a bit tempted to fail hard here rather than fall back to polling. I'm kinda okay with errors that happen on startup and I think it's better to force folks to set their stuff up correctly. We can always relax it later but if we do this now, I doubt we can enforce it later